### PR TITLE
Fix Broken and old links in VZV and VZE

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/16768

## Testing
**URL to test:** 
[VZV](https://deploy-preview-1431--atd-vzv-staging.netlify.app/)  and [VZE](https://deploy-preview-1431--atd-vze-staging.netlify.app/#/crashes)

**Steps to test:**
1. First test the VZV, check that the link to the CR3 police manual links to the 2024 version now. 
2. Also test that the link at the footer to our website works.
3. Now test the VZE deploy, test that the link to our website at the bottom works.

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-vze/src/containers/DefaultLayout/DefaultFooter.js [hunk 1]
```diff
@@ -20,8 +20,8 @@ class DefaultFooter extends Component {
         </span>
         <span className="ml-auto">
           Powered by{" "}
-          <a href="https://transportation.austintexas.io/about/">
-            ATD Data & Technology Services
```
Comment (chiaberry) [inline atd-vze/src/containers/DefaultLayout/DefaultFooter.js:24]: 😬

**Review by chiaberry — APPROVED** [review]

**Review by mddilley — APPROVED** [review]
Thanks for reviving these links! Test steps check out. 🙌 :shipit:

File: atd-vze/src/containers/DefaultLayout/DefaultFooter.js (hunk 1 above)
Comment (roseeichelmann) [reply atd-vze/src/containers/DefaultLayout/DefaultFooter.js:24]: :wave:

**Review by frankhereford — APPROVED** [review]
# ⛓️ 🧹 👍 🚢 🙏

**Review by johnclary — APPROVED** [review]
these links were very broken and old. thank you! 🚢 🚢 🚢 🚢 🚢 🚢 🚢

**Pull request merged**